lib/logthrdest/logthrdestdrv.c

```diff
@@ -1108,8 +1109,9 @@ log_threaded_dest_driver_init_method(LogPipe *s)
   if (self->time_reopen == -1)
     self->time_reopen = cfg->time_reopen;
 
-  self->shared_seq_num = GPOINTER_TO_INT(cfg_persist_config_fetch(cfg,
-                                         _format_seqnum_persist_name(self)));
+  value = cfg_persist_config_fetch(cfg, _format_seqnum_persist_name(self));
```

Comment:
Again, make ```value``` inline declared. Maybe a better name would make it even better. (E.g. persisted_value)

Reply:
fixed 👍